app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @post = Post.find(params[:id])
+  end
+end
```

Comment:
- [ ] You have two actions in the PostsController :blush: That's great but you should have two views for these two actions :thinking: As you can see in your `app/views/posts` folder that there is only one view which is **show.html.erb** you should have another view by the name of **index.html.erb** :blush: for your index action :+1: